src/k8s-engine/wsl.ts

```diff
@@ -1306,6 +1309,50 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     await this.execCommand('/usr/sbin/update-ca-certificates');
   }
 
+  /**
+   * Run provisioning scripts; this is done after init is started.
+   */
+  protected async runProvisioningScripts() {
+    const provisioningPath = path.join(paths.config, 'provisioning');
+
+    await fs.promises.mkdir(provisioningPath, { recursive: true });
+    await Promise.all([
+      (async() => {
+        // Write out the readme file.
+        const ReadmePath = path.join(provisioningPath, 'README');
+
+        try {
+          await fs.promises.access(ReadmePath, fs.constants.F_OK);
+        } catch {
+          const contents = `
+            Any files named '*.start' in this directory will be executed
+            sequentially on Rancher Desktop startup, before the main services.
+            Files are processed in lexical order, and will delay startup until
+            they are complete. Similaryly, any files named '*.stop' will be
+            executed on shutdown, after the main services have exited.
```

Comment:
README needs an EOL at end of text.

Reply:
Added in commit dc877bc39d7fcc9698103964743e8dc0957eb6be.